compiler/src/iree/compiler/Codegen/Dialect/GPU/Transforms/Passes.td

```diff
@@ -15,23 +15,6 @@ def CombineBarrierRegionsPass :
   let dependentDialects = ["::mlir::iree_compiler::IREE::GPU::IREEGPUDialect"];
 }
 
-def ConcretizeMmaShapesPass :
-    InterfacePass<"iree-gpu-concretize-mma-shapes", "mlir::FunctionOpInterface"> {
```

Comment:
hmmm I would have thought this one stays because it's `multi_mma` -> `multi_mma`.

Reply:
Sounds good, I'll move this back.

Reply:
updated